### DISCUSSION TITLE
Can the MOORDYN compute a structure where the mooring line consists of several segments of different materials?

### DISCUSSION THREAD

**Discussion by haoshuning** [OpenFAST/openfast#2105] [General]
Hi,everyone.
  I've recently been analyzing the mooring system of IEA 15MW, and I would like to ask if the moordyn module can divide the mooring line into several sections of cable of different materials for calculation? Because when i see the moordyn, I only see one place to enter the line type, just as the picture shows. If the moordyn can do it , how should I implement this? Or can other mooring modules of OPENFAST do it?
  THANK YOU FOR YOUR ANSWER!!!! THANK YOU VERY MUCH!!
![3](https://github.com/OpenFAST/openfast/assets/160987005/d1ad6e11-340c-4f58-b40e-9baa2316e408)

**Comment by jjonkman** [comment]
Dear @haoshuning,

Yes, you can model segmented mooring lines with different mooring line properties in each segment.  You can specify multiple line types (with different `TypeName`s) to specify the different properties of each line type.  See the online MoorDyn documentation for more information: https://moordyn.readthedocs.io/en/latest/inputs.html.

Best regards,
